src/components/browse/singleDatasetView/singleDatasetViewAccordion/tables/ExperimentsTable.ts

```diff
@@ -0,0 +1,60 @@
+import { useState } from "react";
+import { datasetEmbeddedModel } from "../../../../../models/dataset";
+import { TransposeTableForHTML } from "../../../../../utils/utils";
+
```

Comment:
rename the file to follow camelCase naming standards

Reply:
Done!

Reply:
Done for real this time